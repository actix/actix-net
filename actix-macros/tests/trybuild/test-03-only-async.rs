#[actix_rt::test]
fn my_test() {
    futures_util::future::ready(()).await
}

fn main() {}
