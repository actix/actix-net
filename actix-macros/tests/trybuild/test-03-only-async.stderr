error: the async keyword is missing from the function declaration
 --> $DIR/test-03-only-async.rs:2:1
  |
2 | fn my_test() {
  | ^^
